scripts/cmscp.py

```diff
@@ -1134,6 +1135,7 @@ def update_exit_info(exit_info, exit_code, exit_msg = None, force = False):
                              'CRAB_StageoutPolicy', \
                              'CRAB_Destination', \
                              'CRAB_Dest', \
+                             'CRAB_Final_Dest', \
```

Comment:
Since we use in the CRAB configuration the parameter `outLFNDirBase` maybe here we can use `CRAB_OutLFNDir` and `CRAB_OutTempLFNDir` (or `CRAB_TempOutLFNDir`, not sure which one is more correct)?